config/gail_config.yaml

```diff
@@ -108,3 +108,23 @@ Hallway:
             gamma: 0.99
             encoding_size: 128
             demo_path: demos/ExpertHallway.demo
+
+FoodCollector:
+    batch_size: 64
+    summary_freq: 1000
+    max_steps: 5.0e4
+    use_recurrent: false
+    hidden_units: 128
+    learning_rate: 3.0e-4
+    num_layers: 2
+    sequence_length: 32
+    reward_signals:
+        gail:
+            strength: 0.1
+            gamma: 0.99
+            encoding_size: 128
+            demo_path: demos/ExpertFood.demo
+    behavioral_cloning:
+        demo_path: ./demos/ExpertFood.demo
```

Comment:
nit: inconsistent `./` between `behavioral_cloning.demo_path` and `gail.demo_path`